messages/en.json

```
@@ -2388,6 +2388,7 @@
     "createSelected": "Preview requirements in import ({count})",
     "createConfirm": "{count, plural, one {Create # requirement in requirement area {area}?} other {Create # requirements in requirement area {area}?}}",
     "createError": "Failed to create requirements",
+    "generationFailed": "Generation failed",
```

Biome (2.5.1)

[error] The key generationFailed was already declared.

(lint/suspicious/noDuplicateObjectKeys)